addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -41,37 +41,37 @@ class IrQWeb(models.AbstractModel):
 
     def _compile_node(self, el, compile_context, indent):
```

Comment:
I am closing https://github.com/odoo/odoo/pull/202780 which is a duplicate of your fix (doing it the "wrong" way, like you first did). Can you add the opw-XXX it mentioned in your own commit message here? :pray: